lib/linter-eslint.js

```diff
@@ -138,6 +143,11 @@ export default {
       try {
         results = JSON.parse(output).results
         messages = results[0].messages || []
+        // clobber ignored message based on settings
+        if (messages.length === 1 && !!~messages[0].message.indexOf('ignored because of your .eslintignore file') &&
```

Comment:
Sure, We could match on the whole string... Would you prefer that? E.g. :
```js
 messages[0].message === 'File ignored because of your .eslintignore file. Use --no-ignore to override.'
```